src/Sql/dbo/Tables/Secret.sql

```diff
@@ -9,7 +9,7 @@ CREATE TABLE [dbo].[Secret]
     [RevisionDate] DATETIME2(7) NOT NULL,
     [DeletedDate] DATETIME2(7) NULL,
     CONSTRAINT [PK_Secret] PRIMARY KEY CLUSTERED ([Id] ASC),
-    CONSTRAINT [FK_Secret_OrganizationId] FOREIGN KEY ([OrganizationId]) REFERENCES [dbo].[Organization]([Id]) ON DELETE CASCADE
+    CONSTRAINT [FK_Secret_OrganizationId] FOREIGN KEY ([OrganizationId]) REFERENCES [dbo].[Organization]([Id]) ON DELETE NO ACTION
```

Comment:
There is no need to specify the action if there is none.
```suggestion
    CONSTRAINT [FK_Secret_OrganizationId] FOREIGN KEY ([OrganizationId]) REFERENCES [dbo].[Organization]([Id])
```

Reply:
makes sense